src/plugins/intel_npu/src/al/include/sync_infer_request.hpp

```diff
@@ -22,7 +22,7 @@ namespace intel_npu {
  */
 class SyncInferRequest : public ov::IInferRequest {
 public:
-    explicit SyncInferRequest(const std::shared_ptr<const ICompiledModel>& compiledModel);
+    explicit SyncInferRequest(std::shared_ptr<const ICompiledModel> compiledModel);
```

Comment:
This change breaks IMD backend build. Is it worth an integration hiccup?

Reply:
I also updated the IMD backend and started a full validation job.